src/core/core.scale.js

```diff
@@ -857,6 +857,11 @@ module.exports = function(Chart) {
 
 					var label = itemToDraw.label;
 					if (helpers.isArray(label)) {
+						if (label.length % 2 === 0) {
```

Comment:
what's the reason for this if? If I understand correctly, `-tickFont.size * label.length  2` is half the height of the label so we should just move up by that amount. I think this is working around simply setting a different text baseline that would make the code cleaner

Reply:
The reason for that if is to check if there is an even amount of labels. If so then the labels get centered evenly since it would be a different calculation when compared to an odd amount of labels. I think you are meaning `-tickFont.size * label.length / 2`? I'm fairly new to the project but I think setting a different baseline could work as well like you said. Thank you for the review.

Reply:
I also don't understand why we need to handle odd/even amount of lines differently and based on your screenshot, it doesn't seem correct when there is 2 lines (I would expect the text centered on the bar):

![image](https://user-images.githubusercontent.com/3874900/35981422-80b90154-0ced-11e8-9bf9-cfb28c4dd00f.png)